tests/fixtures/Middleware/proxy-ticket-test.yml

```diff
@@ -0,0 +1,229 @@
+-
+  ticket: DESKTOP_PT-21-9fp9
+  xml: |
```

Comment:
This xml is repeated. Here, in `tests/fixtures/Requests/request-send-test.yml` and in `tests/fixtures/Requests/response-parse-test.yml`. And may it's going to be needed more times?

It should be a fixture _per se_ and reused when necessary.

On the other hand, it's not really relevant to the test, is it? If it's a temporary work around then it's not a big problem. But then we need a ticket to work on this, it can't stay as is.

Reply:
I think we discussed with @ademarco already about this.
We would like to have in the fixture files all the necessary input data and the expectations.
Obviously, this might lead to some duplicated data.

Reply:
Yes duplicated data is fine but if on the same test we need always the same input then that's not supposed to be part of the fixture and can be moved within the test itself, thus considering it as test setup.

Reply:
Here and in most of the cases xml for responses can be simplified to just ensure we get a createRequestsResponse in response. Like this:

```
     <?xml version='1.0' encoding='UTF-8'?>
      <S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
          <S:Body>
              <ns0:createRequestsResponse xmlns:ns0="http://eu.europa.ec.dgt.epoetry">
              </ns0:createRequestsResponse>
          </S:Body>
      </S:Envelope>
```

But anyway, my point was that it shouldn't be repeated but if you consider that duplicated is ok I rest my case.